ddtrace/appsec/_common_module_patches.py

```diff
@@ -42,6 +46,16 @@ def patch_common_modules():
     global _is_patched
     if _is_patched:
         return
+    # for testing purposes, we need to update is_iast_request_enabled
```

Comment:
nit: could be interesting to have this code in a separate function and call it here? So we can call it from tests directly?